src/main/java/app/service/AdministrativoService.java

```diff
@@ -1,5 +1,43 @@
 package app.service;
 
+
+import app.model.odb.User;
+import app.repositories.RepositorioDeUsuarios;
+
+
 public class AdministrativoService {
+	
+	private static RepositorioDeUsuarios repoUsers = RepositorioDeUsuarios.getInstance();
+
+	public static User obtenerUsuario(String id) {
+		return repoUsers.search(Integer.parseInt(id));
+	}
+
+	/**
+	 * @return the repoUsers
+	 */
+	private static RepositorioDeUsuarios getRepoUsers() {
+		return repoUsers;
+	}
+
+	/**
+	 * @param repoUsers the repoUsers to set
+	 */
+	private static void setRepoUsers(RepositorioDeUsuarios repoUsers) {
+		AdministrativoService.repoUsers = repoUsers;
+	}
 
+	public static List<Movie> obtenerInterseccionListas(User user1, User user2) {
+		Set<Movie> interseccionSet = new HashSet<Movie>();
+		List<MovieList> user1Lists = user1.getLists();
+		for (MovieList ml1:user1Lists){
```

Comment:
Pasa que es una lista dentro de otra lista. No sé si te lo toma así. En el otro PR hago un for que usa el filter también.